src/Controller/Component/ThumbsComponent.php

```diff
@@ -91,6 +91,9 @@ protected function getThumbs(array $ids): ?array
             ]));
             $apiClient = ApiClientProvider::getApiClient();
             $res = $apiClient->get($url, $query);
+            if ($res == null) {
+                return [];
+            }
 
```

Comment:
`(array)$res = $apiClient->get($url, $query);` should do the job (and we avoid an extra path, caused by `if`)